pkg/report/report.go

```diff
@@ -439,17 +449,26 @@ func Generate(ctx context.Context, path string, mrs yara.MatchRules, c malconten
 				syscalls = append(syscalls, sy...)
 			case "cap":
 				caps = append(caps, v)
-			// If we find a rule to override, set that rule's risk score and level
+			// If we find a rule to override, pull in that rule's configuration and update the severity
 			case overrideRule:
 				var overrideSev int
 				if sev, ok := Levels[v]; ok {
 					overrideSev = sev
 				}
-				// Find the behavior for the overridden rule
-				for _, b := range fr.Behaviors {
-					if b.RuleName == overrideRule {
+				// Find the behavior for the overridden (original) rule
+				// Store its behavior in the current behavior and remove the original behavior
+				for i, ob := range fr.Behaviors {
+					if ob.RuleName == overrideRule {
```

Comment:
@tstromberg -- let me know if you think we should pull in any of the original rule's fields (author, URL, etc.).

Reply:
My suggestion: preserve everything in the original rule; just override the severity and add a suffix to the description.

Reply:
Updated in: [`bbe0701` (#481)](https://github.com/chainguard-dev/malcontent/pull/481/commits/bbe07015e121cdf0d4db52791800a11c645b5622)

Example of the description suffix:
```
MED   secrets/bash_history     access .bash_history file, overridden by            .bash_history                     
                               yes_it_is_really_bash                                            
```